package.json

```diff
@@ -120,6 +120,6 @@
     "verify": "npm-run-all build lint test"
   },
   "insights": {
-    "appname": "insights-rbac"
+    "appname": "rbac"
```

Comment:
Does line 2 have to change also?  I've been trying to keep both of these lines in sync.

Reply:
Not really. The second line in this file represents package name, this on the other hand is representing application name in the insights/cloud services project.